changelogs/CHANGELOG-v0.3.1.md

```diff
@@ -0,0 +1,69 @@
+The v0.3.1 release is a patch release which includes some bug fixes and makes
+the ability for bpf enabled applications to receive their maps via bpfd's
+custom CSI plugin default.
+
+This enablement means that most applications can now access their maps **WITHOUT**
+being run as root.
+
+> **WARNING**
```

Comment:
```suggestion
> [!WARNING]
```

Will make this prettier.

Reply:
The older syntax was just deprecated earlier this month: https://github.com/orgs/community/discussions/16925

Reply:
Sweet thanks I didn't know that!